### PR TITLE
Add calculator

### PR DESCRIPTION
**Pull request by JuliCarracedo**
This PR by @JuliCarracedo introduces the style and form of the math magicians' calculator.

### CODE REVIEW

File: src/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    </div>
+    <div className="row">
+      <div className="big-button gray">0</div>
+      <div className="button gray">.</div>
+      <div className="button orange">=</div>
+    </div>
+  </div>
+);
+
+export default Calculator;
```
Comment (Emmyn5600) [inline src/Calculator.js]: - [x] There should be a Calculator.js file inside the components directory.
- [x] The Calculator.js component should be class-based
storage5

- [x] The App.js component should be class-based.
- [x] The App.js(main component) should have the imported the Calculator.js component 

!storage1

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @JuliCarracedo 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
 - The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
-  Git/GitHub flow best practices have been followed.:heavy_check_mark:
- Great design matching with the given design:heavy_check_mark:

Changes required :writing_hand: 
  
   - Please check the comment under the review.

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
+// class Field extends React.Components {
```
Comment (oxenprogrammer) [inline components/Calculator.js]: **Changes Required**
- [x] Let us try not to commit commented out code to our repo. It makes it look unprofessional. You can use `git history` to track your changelog. Let us try to remove these and delete this directory `src/components` as a whole.

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @JuliCarracedo , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. :muscle:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Consider the optional changes nice to have features and only work on them when you have satisfied the required changes_

- [ ] Your design is elegant but let us try to follow the given design as close as possible. Converting UX/UI design into code is definitely one of the roles you will play as a front-end developer.
**Your Design**
cal

**Expected Design**
calculator


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by oxenprogrammer — APPROVED** [review]
Hi @JuliCarracedo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**